src/Build.OM.UnitTests/TransientIO.cs

```diff
@@ -77,7 +77,7 @@ public string GetAbsolutePath(string relative)
             var absolute = Path.GetFullPath(Path.IsPathRooted(relative) ? relative : Path.Combine(tempRoot, relative));
             if (!IsControled(absolute))
             {
-                throw new ArgumentException(nameof(relative));
+                throw new ArgumentException(null, nameof(relative));
```

Comment:
💭 I would prefer to see a better message here if possible

Reply:
It looks like the code is checking that the relative path is indeed relative to the RootFolder. So I supposed the error should be something like:
`The path is invalid. Either the path is an absolute path that is not rooted to the RootFolder, or just an invalid path`.

Reply:
`Either the path is invalid or it is not rooted to RootFolder.`
? Bit more concise.